test/ApplePlatformsIntegrationTestApp/Podfile

```diff
@@ -0,0 +1,7 @@
+source 'https://cdn.cocoapods.org/'
+
+target 'ApplePlatformsIntegrationTestApp' do
```

Comment:
I don't really know what I'm asking, but... does this mean the Xcode project is generated?  If so, should it be removed from the PR?

Reply:
No, this points _at_ a target inside the `xcodeproj` document. Having said that, I excluded all other files that are generated from the repo except for the `xcworkspace` document, I’ll remove that too.